tests/conftest.py

```diff
@@ -58,3 +58,142 @@ def app_config(app_config):
+        "contact": "info@inveniosoftware.org",
+        "community": {
+            "primary": "Maincom",
+            "secondary": ["Subcom One", "Subcom Two"]
+        },
+        "resource_type": {
+            "type": "image",
+            "subtype": "photo"
+        },
+        "identifiers": [{
+            "identifier": "10.5281/zenodo.9999999",
+            "scheme": "DOI"
+        }, {
+            "identifier": "9999.99999",
+            "scheme": "arXiv"
+        }],
+        "creators": [{
+            "name": "Julio Cesar",
+            "type": "Personal",
+            "given_name": "Julio",
+            "family_name": "Cesar",
+            "identifiers": [{
+                "identifier": "9999-9999-9999-9999",
+                "scheme": "Orcid"
+            }],
+            "affiliations": [{
+                "name": "Entity One",
+                "identifier": "entity-one",
+                "scheme": "entity-id-scheme"
+            }]
+        }],
+        "contributors": [{
+            "name": "Maximo Decimo Meridio",
+            "type": "Personal",
+            "given_name": "Maximo",
+            "family_name": "Decimo Meridio",
+            "identifiers": [{
+                "identifier": "9999-9999-9999-9998",
+                "scheme": "Orcid"
+            }],
+            "affiliations": [{
+                "name": "Entity One",
+                "identifier": "entity-one",
+                "scheme": "entity-id-scheme"
+            }],
+            "role": "RightsHolder"
+        }],
+        "titles": [{
+            "title": "A Romans story",
+            "type": "Other",
+            "lang": "eng"
+        }],
+        "descriptions": [{
+            "description": "A story on how Julio Cesar relates to Gladiator.",
+            "type": "Abstract",
+            "lang": "eng"
+        }],
+        "publication_date": "2020-06-01",
+        "licenses": [{
+            "license": "Copyright Maximo Decimo Meridio 2020. Long statement",
+            "uri": "https://opensource.org/licenses/BSD-3-Clause",
+            "identifier": "BSD-3",
+            "scheme": "BSD-3",
+        }],
+        "subjects": [{
+            "subject": "Romans",
+            "identifier": "subj-1",
+            "scheme": "no-scheme"
+        }],
+        "language": "eng",
+        "dates": [{
+            "start": "2020-06-01",
+            "end":  "2021-06-01",
+            "description": "Random test date",
+            "type": "Other"
+        }],
+        "version": "v0.0.1",
+        "related_identifiers": [{
+            "identifier": "10.5281/zenodo.9999988",
+            "scheme": "DOI",
+            "relation_type": "Requires",
+            "resource_type": {
+                "type": "image",
+                "subtype": "photo"
+            }
+        }],
+        "references": [{
+            "reference_string": "Reference to something et al.",
+            "identifier": "9999.99988",
+            "scheme": "GRID"
+        }],
```

Comment:
What are references again (maybe @lnielsen) ? I know they are not part of Datacite, but I also saw that Datacite has a "references" relation type in related identifiers...

Reply:
References is more like other records, journals, etc. that do not have an identifier. It's basically what would go in the "reference" section of a journal/paper.